Gemfile

```diff
@@ -2,45 +2,47 @@ source 'https://rubygems.org'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.7.1'
-# Use PostgreSQL as the database for Active Record
-# gem 'pg'
+gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
```

Comment:
There was a reason I disabled the database :P Heroku does not support sqlite, and since for now we don't need a db, I left postgres commented out as well.